corehq/apps/locations/ucr_expressions.py

```diff
@@ -62,3 +64,48 @@ def location_parent_id(spec, context):
         }
     }
     return ExpressionFactory.from_spec(spec, context)
+
+
+class AncestorLocationExpression(JsonObject):
+    """
+    For a given location id and location type, this expression returns the ancestor of the location that is the
+    given type.
+    If no such location exists, return None.
+    e.g. (boston.location_id, "state") => massachusetts.to_json()
```

Comment:
That's true. I think my preference is to keep this here because someone exploring the code might not know that the readme exists, and I also don't anticipate the behavior or this expression to change much if at all.